chain-impl-mockchain/src/testing/builders/tx_cert_builder.rs

```diff
@@ -211,6 +211,10 @@ pub fn tally_sign(
 
             TallyProof::Public { id, signature }
         }
+        PayloadType::Private => {
+            //TODO: how to sign this?
+            unimplemented!("Need to implement signature for private votes")
+        }
```

Comment:
We need to discuss what should the flow of signing this be.

Reply:
Should be signed with the sender key. What is the problem here actually?

Reply:
@danielSanchezQ no matter what we decide regarding the committee keys, the signature of certificates will always require ed25519 keys, so in this case the flow is the same, just the returned TallyProof is private.